packages/components/psammead-content-shift-blocker/src/index.stories.jsx

```diff
@@ -0,0 +1,173 @@
+          {renderParagraphs({ script, service })}
+          <Gallery script={script} service={service} />
+          {renderParagraphs({ script, service })}
+          <Gallery script={script} service={service} />
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          <Gallery script={script} service={service} />
+          <Gallery script={script} service={service} />
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+        </Wrapper>
+      );
+    },
+    { notes },
+  )
+  .add(
+    'multiple staggered',
+    ({ script, service }) => {
+      return (
+        <Wrapper>
+          <Headline>The Daily Doggo</Headline>
+          {renderParagraphs({ script, service })}
+          <Gallery delay={4000} script={script} service={service} />
+          {renderParagraphs({ script, service })}
+          <Gallery delay={3000} script={script} service={service} />
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          <Gallery delay={2000} script={script} service={service} />
+          <Gallery delay={1000} script={script} service={service} />
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+          {renderParagraphs({ script, service })}
+        </Wrapper>
+      );
+    },
+    { notes },
+  )
+  .add(
+    'nothing but cats',
+    ({ script, service }) => {
+      return (
+        <Wrapper>
+          <Gallery delay={1000} script={script} service={service} />
+          <Gallery delay={4000} script={script} service={service} />
+          <Gallery delay={9000} script={script} service={service} />
+          <Gallery delay={4000} script={script} service={service} />
+          <Gallery delay={7000} script={script} service={service} />
+          <Gallery delay={2000} script={script} service={service} />
+          <Gallery delay={3000} script={script} service={service} />
+          <Gallery delay={2000} script={script} service={service} />
+          <Gallery delay={5000} script={script} service={service} />
+          <Gallery delay={9000} script={script} service={service} />
+          <Gallery delay={8000} script={script} service={service} />
+          <Gallery delay={2000} script={script} service={service} />
+          <Gallery delay={6000} script={script} service={service} />
+          <Gallery delay={8000} script={script} service={service} />
+          <Gallery delay={3000} script={script} service={service} />
+          <Gallery delay={2000} script={script} service={service} />
+        </Wrapper>
+      );
+    },
+    { notes },
```

Comment:
I agree with Sareh on this one. I don't see the value in a ChromaticQA test especially because there is no visible UI and we have unit tests for the functionality. ChromaticQA has in the past flagged up IE javascript errors so if we think there's value in that I could render a div with a background colour that this component wraps as a ChromaticQA test

Reply:
I'm happy with this as you say @jroebu14 given there are unit tests